test/scripts/products/h22-validate-that-rate-limit-service-is-working-as-expected/h22-validate-that-rate-limit-service-is-working-as-expected.go

```diff
@@ -594,3 +628,50 @@ func newThreescaleClient(installation *integreatlyv1alpha1.RHMI, accessToken str
 
 	return threescale.NewThreeScaleClient(httpc, installation.Spec.RoutingSubdomain)
 }
+
+// mock3scaleAPI used to simulate an API while creation of one failed but we still want to clean up resources
+func mock3scaleAPI(ctx context.Context, client k8sclient.Client, namespacePrefix string) *threescaleAPI {
+	namespace := fmt.Sprintf("%soperator", namespacePrefix)
+	tsNamespace := fmt.Sprintf("%s3scale", namespacePrefix)
+
+	s := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "system-seed",
+		},
+	}
+	err := client.Get(ctx, k8sclient.ObjectKey{Name: s.Name, Namespace: tsNamespace}, s)
+	if err != nil {
+		os.Exit(1)
+	}
+	accessToken := string(s.Data["ADMIN_ACCESS_TOKEN"])
+	fmt.Printf("  🔑  Found access token: %s\n", accessToken)
+	installation, err := resources.GetRhmiCr(client, ctx, namespace, logger.NewLogger())
```

Comment:
`resources.GetRhmiCr()` has been [deprecated](https://github.com/carlkyrillos/integreatly-operator/blob/master/pkg/resources/rhmi.go#L14) and `rhmi.GetRhmiCr()` should be used [instead](https://github.com/carlkyrillos/integreatly-operator/blob/master/pkg/resources/rhmi/rhmi.go#L13). The functionality is the exact same so switching it shouldn't require any further changes.
```suggestion
	installation, err := rhmi.GetRhmiCr(client, ctx, namespace, logger.NewLogger())
```

Reply:
nicely spotted. Replaced